creator-node/src/ipfsClient.js

```diff
@@ -22,4 +26,74 @@ async function logIpfsPeerIds () {
   logger.info(`Current IPFS Peer ID (using latest version of ipfs client): ${JSON.stringify(identityLatest, null, 2)}`)
 }
 
-module.exports = { ipfs, ipfsLatest, logIpfsPeerIds }
+/**
+ * Wrapper to ipfs.addToFs() -- This wrapper allows for ipfs calls to be made async or synchronous.
+ * @param {Object} ipfs ipfs library -- can be two different versions (see ipfsClient.js)
+ * @param {string|string[]} buffers
+ * @param {Object?} config ipfs add config options
+ * @param {string?} caller the name of the caller fn
+ * @param {boolean?} isMetadata if the input data is of a metadata type. if so, always synchronously add to ipfs
+ * @returns {string|string[]} hashes
+ */
+async function ipfsAddWrapper (ipfs, buffers, config = {}, caller = '', isMetadata = false) {
```

Comment:
passing the name of the caller seems like a very anti patterny thing to do.
https://stackoverflow.com/questions/14172455/get-name-and-line-of-calling-function-in-node-js

at the very least, maybe we could call it context and accept the params here as a config object (rather than multiple args) so it's always optional

Reply:
going to pass in the req.logContext instead~